api/stream-relay/index.js

```diff
@@ -1808,26 +2016,22 @@ export const wrap = dgram => {
      */
     async _onMessage (data, { port, address }) {
       const packet = Packet.decode(data)
-      if (!packet || packet.version < VERSION) return
+      if (!packet || packet.version !== VERSION) return
```

Comment:
does it mean that any packet of message that is not compatible with current version will be ignored? this can make it hard for people who don't update their apps frequently have less peers or no peers.